frontend/calangute-beach/index.html

```
@@ -0,0 +1,365 @@
+            </div>
+        </div>
+        <div class="footer-bottom">
+            <p>&copy; 2026 Incredible India Explorer. All rights reserved.</p>
+        </div>
+    </footer>
+
+    <!-- Lightbox Modal -->
+    <div class="lightbox-modal" id="lightbox-modal" role="dialog" aria-modal="true" aria-label="Expanded image viewer with attribution">
+        <button class="lightbox-close" id="lightbox-close-btn" aria-label="Close image modal">&times;</button>
+        <div class="lightbox-content">
+            <img src="" id="lightbox-img" alt="">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)